### ISSUE TITLE
Theme: Default to "Light" instead of "Set by battery saver"

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/talk-android#681]
Currently the theme switches every evening at 18:00 to dark mode, which is a bit strange and unexpected. (Also, some things are still to be fixed in dark mode, but that’s separate. :)

The theme should ideally default to "Light". With "Dark" next in the list, and "Set by battery saver" being last in the list.

-----

@mario if you point out where in the code that would have to be done, this would be a very nice good first issue. :)

**Comment by mario** [comment]
It's actually "Follow system" or "Battery saver" depending on which version of Android you're running. 

Is there a reason why we should not follow what the system does/what you configured in the OS settings by default?

**Comment by jancborchardt** [comment]
Right – so "Follow system" means to follow the system setting on Light or Dark mode? Cause there indeed it would be expected to change the theme.

For "Battery saver" systems however, it seems a bit strange to change the theme to dark. Especially cause the dark theme is not "true black" which won’t really change energy on the specific screens, will it? (Also haven’t seen other apps with light and dark themes do that for energy saver mode, but that might just be selection bias.)

So basically, I’d suggest to:
- Default to "Follow system" if the Android version supports "Follow system" on theming
- Default to "Light" otherwise

Does that make sense?

**Comment by mario** [comment]
While I don't have a super strong opinion on this, the reason why I've done it the way I did is due to Google recommendations which you can see here:

https://developer.android.com/guide/topics/ui/look-and-feel/darktheme

To quote:

> When running on devices running Android 9 or earlier, the recommended theme options are:

    Light
    Dark
    Set by Battery Saver (the recommended default option)

When running on Android 10 (API level 29) and higher, the recommended options are different, to allow the user to override the system default:

    Light
    Dark
    System default (the recommended default option)

**Comment by mario** [comment]
Let me know if you still prefer Android 9 and earlier to have "Light" as default after reading Google recommendations.

**Comment by jancborchardt** [comment]
Thanks a lot for the info and the link to it! :)

Good call to do it as per recommendations. However just for the Android 9 or earlier I’d say let’s do Light as default, as:
- It’s not expected to change theme based on battery state
- We have some detail issues to work out with the dark theme (separate issue though)
- Others do this as well

Thanks a lot! :)

**Comment by mario** [comment]
This is fixed, but not yet merged.
